### DISCUSSION TITLE
2 "light" accessories created for 1 ESP32?

### DISCUSSION THREAD

**Discussion by ssstraub** [HomeSpan/HomeSpan#311] [Q&A] [answered]
I'm brand new to HomeSpan, but already got it working using the example sketch "05-Working LED".

One thing I don't understand is why the setup process on the HomeKit side created two accessories in the Home app? There are 2 light bulbs now. One doesn't do anything and the other controls the LED on pin 17. Why is there a redundant light object? 

I didn't customize literally anything in the example, just connected, compiled, added to Home app. 

Here's what I found in serial monitor. It seems like there are two accessories for reasons I don't understand?
<img width="450" alt="image" src="https://user-images.githubusercontent.com/10637893/166983562-94a44a78-974c-4b44-905e-7afb4abbb1a6.png">

**Comment by HomeSpan** [comment, marked as answer]
The example 5 is designed for two lights - one is controlled on pin 16 and one on pin 17.

**Reply by ssstraub** [reply]
Doh, I see in the comments on that example that you are correct. Sorry for missing that!
